extensions/ql-vscode/test/vscode-tests/no-workspace/databases/db-panel-selection.test.ts

```diff
@@ -50,101 +51,150 @@ describe("db panel selection", () => {
     await remove(workspaceStoragePath);
   });
 
-  it("should mark selected remote db list as selected", async () => {
-    const dbConfig: DbConfig = createDbConfig({
-      remoteLists: [
-        {
-          name: "my-list-1",
-          repositories: ["owner1/repo1", "owner1/repo2"],
+  describe("when controller repo is not set", () => {
```

Comment:
Apologies for the fly-by comment, but should these tests be in `db-panel-rendering.test`? This file contains tests around the selection functionality.

Reply:
Thanks for noticing! I've moved the test to the rendering test file.